### PR TITLE
2478 add streets table from datagov

### PR DESCRIPTION
**Pull request by ziv17**

### CODE REVIEW

File: anyway/parsers/streets.py
Comment (tkalir) [inline anyway/parsers/streets.py:1]: Hi, looks good. 
Are the changes in infographics_data_cache_updater.py related to this code? 
Also, this table gets data only from CBS so we can be sure we won't have in the table the same street twitce under slightly different names, right?

Comment (ziv17) [reply anyway/parsers/streets.py:1]: Until now we had 21K streets. The list from CBS has 61K streets. So we enter to streets cache only those streets that has accidents.

**Review by tkalir — APPROVED** [review]

File: anyway/parsers/streets.py
Comment (atalyaalon) [inline anyway/parsers/streets.py:12]: @ziv17 this limit is too small (and is it necessary to add a limit? perhaps we can just query all data?) since today there are 61455 records in the table, and I assume we can reach a time, at some point where 100000 is not enough.

**Comment by atalyaalon** [comment]
@ziv17 Great work! It's already merged to dev but not to master yet: I'm waiting for the CBS final answer that our method is correct for the merge.
In addition, before merging to production I suggest you also remove the code that updates the streets table from CBS data (this code is in cbs anyway/parsers/cbs/executor.py) - I opened an issue for this: https://github.com/data-for-change/anyway/issues/2674

However, let's wait for the final answer of the CBS and then implement this last part

File: anyway/parsers/streets.py
Comment (ziv17) [reply anyway/parsers/streets.py:12]: The default limit is 32k, so we must supply a limit. I did not find a way to work with pagination. I will set the limit to 1M.

**Pull request merged**
